internal/acceptance/acceptance.go

```diff
@@ -45,6 +45,26 @@ func init() {
 	}
 }
 
+func TestAccPreCheck(t *testing.T) {
```

Comment:
When the SSH key resources were moved out of the `equinix` package, the `TestAccPreCheckForMetal` function was added.  That function only checks for Metal-related env vars.  This one is a drop-in replacement for the private `testAccPreCheck` function in order to keep the diff smaller; we can revisit this implementation in a future refactor.